File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n = nums.size();
        int arr[2];
        for(int i=0 ; i<n-1 ; i++){
            for(int j=i+1 ; j<n ; j++){
                if(nums[i] + nums[j] == target){
                    return {i,j};
                }
            }
        }
        return {-1,-1};
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        dic={}
        n=len(nums)
        for i in range (0,n):
            req=target-nums[i]
            if req in dic:
                return (dic[req],i)
            dic[nums[i]]=i
        return []
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        